### ISSUE TITLE
Ark Ascended

### ISSUE THREAD

**Issue by Niocas** [arkutils/Purlovia#156]
Discussions are not enable, so I wanted to ask, if this project gonna be continued for Ark Ascended?

**Comment by cadon** [comment]
We'd like to. The new engine makes it difficult to get the needed species data, though. The currently used process to get the data won't work with the new engine anymore. The best would be to adjust the current process to the new engine, but that may be very complex and difficult, so a manual data collection could be necessary. We're not yet sure how to organize that. We'll probably talk about that in our discord server if ASA is available and we can evaluate the data retreiving process better.
Since the taming and breeding mechanic proably won't be changed it could be the currently used data will be also valid for ASA or that just some minor adjustments are needed.

**Comment by coldino** [comment]
Yup, that is a good way of putting it - we'd like to continue for sure, but extracting data from UE5 games is a lot harder than UE4 ones as the metadata is not included in the assets themselves any longer.

So as Cad says, from day one we'll likely make a new Obelisk and follow a very similar format but do it manually.

If you are interested at all, please join the [ASB Discord](https://discord.gg/qCYYbQK) so we can keep in touch.
